### PR TITLE
Add a place-name-resolver tool

### PR DESCRIPTION
**Pull request by pradh**
This is for use in importing OECD. See README for more details.

### CODE REVIEW

File: tools/place_name_resolver/testdata/input_basic.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name,unused
+India,A not so small country
+Mumbai,A city that used to have a better name
+Maharashtra,Literally a large state
```
Comment (tjann) [inline tools/place_name_resolver/testdata/input_basic.csv:4]: Love it.

File: tools/place_name_resolver/resolver.go [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+	// Expanded name used for geocoding, one per row.
+	extNames []string
+
+	// Map from node-id to row index.
+	node2row map[string]int
+
+	// Various indexes. -1 if missing.
+	nidIdx  int
+	cipIdx  int
```
Comment (tjann) [inline tools/place_name_resolver/resolver.go:87]: Maybe specify what nid (node id?) and cip (contained in place?) are?

File: tools/place_name_resolver/resolver.go [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+		return name, nil
+	}
+
+	idx, ok := tinfo.node2row[cipRef]
+	if !ok {
+		log.Printf("ERROR: Unresolved 'containedInPlace' ref %s in Node %s, skipping.", cipRef, nodeId)
+		return name, nil
+	}
+	if idx >= len(tinfo.rows) {
+		return "", fmt.Errorf("Out of range %d vs. %d", idx, len(tinfo.rows))
```
Comment (tjann) [inline tools/place_name_resolver/resolver.go:112]: When would this happen?

Comment (pradh) [reply tools/place_name_resolver/resolver.go:115]: Shouldn't really. Added a log.Fatal.

File: tools/place_name_resolver/testdata/input_basic.csv (hunk 1 above)
Comment (pradh) [reply tools/place_name_resolver/testdata/input_basic.csv:4]: :)

File: tools/place_name_resolver/resolver.go (hunk 2 above)
Comment (pradh) [reply tools/place_name_resolver/resolver.go]: Improved comments.

**Pull request merged**